pylops_mpi/DistributedArray.py

```diff
@@ -96,8 +128,9 @@ class DistributedArray:
         Type of elements in input array. Defaults to ``numpy.float64``.
     """
 
+    # TODO: Type Annotation for base_comm without NCCL import
     def __init__(self, global_shape: Union[Tuple, Integral],
-                 base_comm: Optional[MPI.Comm] = MPI.COMM_WORLD,
+                 base_comm=MPI.COMM_WORLD,
```

Comment:
look at this example for how to add type annotation of a package that may not be present: https://github.com/PyLops/pylops/blob/44bae08e43c34516b71ac08e4e445e5835ddcc47/pylops/waveeqprocessing/twoway.py#L14